app/javascript/application.js

```
@@ -0,0 +1,25 @@
+import { Turbo } from '@hotwired/turbo-rails';
+Turbo.start();
+
+import Vue from 'vue/dist/vue.esm';
```

ESLint

[error] 'Vue' is defined but never used.

(no-unused-vars)